Packaging/switch/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DEVKITPRO="${DEVKITPRO:-/opt/devkitpro}"
+
+declare -r DIR="$(dirname "${BASH_SOURCE[0]}")"
+cd "$DIR"
+
+main() {
+	prepare_devkitpro
+	set -x
+	install_deps
+	build
+}
+
+build() {
+	mkdir -p ../../build
+	cd ../../build
+	rm -f CMakeCache.txt
+	DEVKITPRO="$DEVKITPRO" cmake .. -DBINARY_RELEASE=ON -DNONET=ON \
+		-DCMAKE_TOOLCHAIN_FILE=../CMake/switch/devkita64-libnx.cmake
+	DEVKITPRO="$DEVKITPRO" make -j "$(nproc)"
```

Comment:
Nproc is Linux only

Reply:
How about `getconf _NPROCESSORS_ONLN`? It's Mac-compatible and hopefully works on msys2 windows as well.